pycparser/c_parser.py

```diff
@@ -22,9 +22,12 @@ def __init__(
             lex_optimize=True,
             lextab='pycparser.lextab',
             yacc_optimize=True,
-            yacctab='pycparser.yacctab',
+            yacctab=None,
```

Comment:
The current yacctab name is "known" in other places in pycparser - this is a subtle point especially w.r.t. installations that won't need to regenerate it. I'd leave the default yacctab name as it was before